examples/borer-circe-migration/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.jetbrains" % "sbt-ide-settings" % "1.1.0")
```

Comment:
Hmm does it make sense to add it to the sbt plugins of the root project?

Reply:
It's only used for `idePackagePrefix`, right?

Reply:
Calling it *only* is an understatement

Reply:
> Hmm does it make sense to add it to the sbt plugins of the root project?

I tried, it doesn't work well with project matrix.

Reply:
> Calling it only is an understatement

Ok, for clarity coz I haven't used it before &mdash; the effect of the plugin on the project is specifically that you can skip the folders (`org/virtuslab/example`) in examples module, right?

Reply:
Or are there any other benefits? I'd be concerned about making the examples confusing for the users who don't know `sbt-ide-settings`, let alone those who don't use IntelliJ (although the latter is rather a small minority)

Reply:
> the effect of the plugin on the project is specifically that you can skip the folders
Yes, this is the main effect

Reply:
Removing the plugin is not an issue. I agree that it may be confusing for some.

Reply:
Yeah... I'd rather refrain from non-standard solutions in the examples :/